finalPrep/modZeroHero.js

```diff
@@ -1,41 +1,70 @@
 // Challenge - See if you can follow the instructions and complete the exercise in under 30 minutes!
 
 // Declare two variables - heroName AND specialAbility - set to strings
-
+var heroName = "Syntax Syd";
+var specialAbility = "resolve syntax errors";
 // Declare two variables - greeting AND catchphrase
+var greeting = ("Hello! I am " + heroName + "!");
```

Comment:
This works - usually not convention to have the `()` wrapped around the string, though 😊